operator/tests/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1091
+
+set -euo pipefail
+
+# Runs operator e2e tests.
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../.. && pwd)"
+
+source "$ROOT/scripts/lib.sh"
+source "$ROOT/scripts/ci/lib.sh"
+
+test_operator_e2e() {
+    info "Starting oerator e2e tests"
+
+    require_environment "KUBECONFIG"
+
+    read -r -d '' kuttl_help <<- _EO_KUTTL_HELP_ || true
+See log and/or kuttl JUnit output for error details.
+Reading operator/tests/TROUBLESHOOTING_E2E_TESTS.md may also be helpful.
+_EO_KUTTL_HELP_
+
+    rm -f FAIL
+
+    info "Fetching kuttl binary"
+    junit_wrap fetch-kuttl \
+               "Download kuttl binary." \
+               "See log for error details." \
+               "make" "-C" "operator" "kuttl"
+
+    info "Deploying operator"
+    junit_wrap deploy-previous-operator \
+               "Deploy previously released version of the operator." \
+               "See log for error details. Reading operator/tests/TROUBLESHOOTING_E2E_TESTS.md may also be helpful." \
```

Comment:
`deploy-previous-via-olm` also uses `kuttl` so I think it's fine to use `${kuttl_help}` here as well.